plugins/net_plugin/net_plugin.cpp

```diff
@@ -892,6 +892,8 @@ namespace eosio {
       std::atomic<size_t>             bytes_sent{0};
       std::atomic<size_t>             block_sync_bytes_received{0};
       std::atomic<size_t>             block_sync_bytes_sent{0};
+      std::chrono::nanoseconds        block_sync_send_start{0ns};    // start of enqueue blocks
+      size_t                          block_sync_send_bytes_sent{0}; // bytes sent in this set of enqueue blocks
```

Comment:
I hate to quibble over naming but `block_sync_bytes_sent` and `block_sync_send_bytes_sent` just blow my mind.
Can we rename this to something like:
`block_sync_total_bytes_sent`
`block_sync_frame_bytes_sent` ?